saleor/data_feeds/google_merchant.py

```diff
@@ -71,8 +71,10 @@ def item_guid(item: ProductVariant):
     return item.sku
 
 
+# It should returns url to item, but right now
+# there is no way to get product_variant url
 def item_link(item: ProductVariant, current_site):
-    return add_domain(current_site.domain, item.get_absolute_url(), not settings.DEBUG)
+    return add_domain(current_site.domain, "", not settings.DEBUG)
```

Comment:
There is not point in doing that if we don't pass a product URL

Reply:
We don't have a product URL in the backend. This is a temporary fix to still keep the "item link" field in the output file so that Google accepts it, but it will probably raise a warning for empty link values. Maybe we should check in the Google Merchant documentation what are requirements for this field.

Anyway, entire feed generation needs to be reworked to include storefront URLs.

Reply:
I linked it previously, it is optional as long it is not an ad.
> https://support.google.com/merchants/answer/6324416?hl=en&ref_topic=6324338

Reply:
If it's optional then we get rid of this field at all.

Reply:
Ok, so I will remove it.